R/maic_unanchored_tte.R

```diff
@@ -16,12 +16,11 @@
 #'   \item time - numeric column, observation time of the \code{status}; unit in days
 #' }
 #'
-#' @return
+#' @return A list of KM plot, analysis table, and diagnostic plot
 #' @importFrom survival Surv survfit coxph cox.zph
 #' @importFrom graphics par axis lines points legend abline
 #' @export
-#'
-#' @examples
+
```

Comment:
Please add an example rather than removing. We're trying to build a better package. We won't be able to pass validation without examples and tests